docs/source/notebooks/api_quickstart.ipynb

```diff
@@ -16,12 +16,13 @@
      "name": "stdout",
```

Comment:
<p>You can use <code>az.plot_ppc</code>  instead</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc3/pull/3911/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='100'/>

Reply:
<p>This will require to add the post_pred dict as <code>posterior_predictive</code>  group in the inference data object. Try something like:</p><p> </p><pre class="ql-syntax" spellcheck="false">idata.concat(az.from_pymc3(posterior_predictive=post_pred), inplace=True)
az.plot_ppc(idata)
# this above will plot the kde for data, to also add the mean as above,
# the ax.axvline has to be kept
</pre><p> </p>

Reply:
<p>At the moment this does not work - <code>pm.sample_posterior_predictive(idata.posterior)</code>  collapses the 3 chains, 1000 draws into 1 chain 3000 draws. <code>arviz.concat</code>  then refuses to concatenate because the dims don't match.</p>

Reply:
does using `keep_size=True` in sample_posterior_predictive solve the issue?

Reply:
I don'T know what I did wrong, but now it works